test/implement_pitch_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class TestPitchImplementation < MiniTest::Test
+  def setup
+    @pitch_note_B4      = MusicTheory::Note.new(pitch: Pitch.new(:B, 4), duration: 1.0, distort: false)
```

Comment:
@bion Do you think injecting a Pitch object is a good way to do this or would it be better to take an array as an argument 

`MusicTheory::Note.new(pitch: [:B, 4], duration: 1.0, distort: false)`

and write a wrapper to turn that into a Pitch object and extract its frequency inside MusicTheory::Note?